go.mod

```diff
@@ -19,6 +19,7 @@ require (
 	github.com/godbus/dbus v4.1.0+incompatible // indirect
 	github.com/gogo/googleapis v1.2.0 // indirect
 	github.com/gogo/protobuf v1.2.1
+	github.com/google/uuid v1.1.1
```

Comment:
Why cant we use the guid package from go-winio?

Reply:
I didn't know it existed :P 

Is that preferred?

Reply:
Yes. Kevin can tell you all the reasons why

Reply:
`github.com/Microsoft/go-winio/pkg/guid` will play more nicely with other Windows mechanisms that expect a GUID. In particular Windows encodes GUIDs to binary in a slightly different way.